### PR TITLE
EPaperDisplay: add rotation property

### PR DESCRIPTION
**Pull request by jepler**
untested, because I don't want to mess my magtag demo up :) but it builds

### CODE REVIEW

**Comment by jepler** [comment]
As predicted, this over-filled the metro_m0_express:
```
Build metro_m0_express for de_DE (clean_build) took 38.73s and failed
make: Entering directory '/home/runner/work/circuitpython/circuitpython/ports/atmel-samd'
Use make V=1, make V=2 or set BUILD_VERBOSE similarly in your environment to increase build verbosity.
QSTR updated

253780 bytes used, -84 bytes free in flash firmware space out of 253696 bytes (247.75kB).
```
I'll see what I can do.

**Comment by ladyada** [comment]
tested non-tall version, rotation works great!

**Review by ladyada — DISMISSED** [review]
tested on hardware, not stylistic review

File: ports/atmel-samd/mpconfigport.h [hunk 1]
```diff
@@ -42,6 +42,7 @@
 #define CIRCUITPY_MCU_FAMILY                        samd21
 #define MICROPY_PY_SYS_PLATFORM                     "Atmel SAMD21"
 #define SPI_FLASH_MAX_BAUDRATE 8000000
+#define MICROPY_ERROR_REPORTING (MICROPY_ERROR_REPORTING_TERSE)
```
Comment (tannewt) [inline ports/atmel-samd/mpconfigport.h:45]: I don't want to switch to errors to TERSE. Good error messages are critical when beginners are beginning to learn to code. It is not something we should sacrifice to have dynamic EInk rotation.

**Review by tannewt — CHANGES_REQUESTED** [review]
Please find another way to make it fit.

File: ports/atmel-samd/mpconfigport.h (hunk 1 above)
Comment (jepler) [reply ports/atmel-samd/mpconfigport.h]: OK, I think I found enough small items to make it fit.

**Review by tannewt — APPROVED** [review]
Wow! Nice work! Thank you so much.

**Pull request merged**